dulwich/tests/test_porcelain.py

```diff
@@ -2167,7 +2178,32 @@ def test_get_untracked_paths_subdir(self):
                     self.repo.open_index(),
                     exclude_ignored=True,
                 )
-            )
+            ),
+        )
+
+    def test_get_untracked_paths_walk(self):
+        os.mkdir(os.path.join(self.repo.path, "dir"))
+        os.mkdir(os.path.join(self.repo.path, "dir/subdir"))
```

Comment:
```suggestion
        os.mkdir(os.path.join(self.repo.path, "dir", "subdir"))
```

Reply:
Seems like there is inconsitency in this test. Sometimes use use separate parts and sometimes you hardcode `/`.